src/lib/zod/index.ts

```diff
@@ -47,6 +47,20 @@ export const createCircleSchemaInput = z
     'Either Protocol name should be filled in or a Protocol should be selected.'
   );
 
+export const createContributionSchemaInput = z
```

Comment:
If we're adding these, it would be nice if we used them in the app...

Reply:
@levity mentioned that these were no longer used and they should even be co-located in the file that uses them, I've tried to just continue the existing pattern and added the few that were missing for my use case, but happy to do it differently if that's preferable. At the time I was not 100% sure what the best way to create the types for the helper functions was, so just went the zod way. Thoughts?

Reply:
I don't think leaving these defs in this file is wrong, since it's at least consistent with the current state.